MMA Removal Utility/SingleTenantSetupInstallation.md

```diff
@@ -0,0 +1,302 @@
+The AzTS MMA Removal Utility solution works in 2 phases:
+1. Discovers VMs and VM Extensions to identify the VMs from which MMA Agent will be removed for which setup requires ***Reader*** access on the scope being configured.
+2. Removes MMA Extensions from the eligible VMs for which setup requires ***Virtual Machine Contributor*** access on the scopes being configured. Scopes Configured can be a Tenant/ManagementGroup(s)/Subscription(s) or both ManagementGroup(s) and Subscription(s).
+
+For executing the above phases, setup requires a [User-Assigned Managed Identity (MI)](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) (identity part of the setup) which has 'Reader and Virtual Machine Contributor' access on target scopes configured.
+
+> _Note:_
+> 1. _For creation of remediation user-assigned MI setup (preferably in a separate new RG), user running this script should have **Owner** access on the subscription in which MI is being created. If the MI is being created in an existing RG, user should have **Contributor** access on the RG._
+> 
+> 2. _For granting remediation user-assigned MI with above mentioned roles on the target scopes, user running this script should have **User Access Administrator (UAA) or Owner** on the configurged scopes. For example, the setup is being configured for a subscription 'X', user should be having UAA role assignment on the subscription 'X' to be able to grant the remediated user-assigned MI with the required permissions._
+>
+
+Before setting up the remediation user-assigned MI, please log in to Azure Account using the following PowerShell command.
+
+``` PowerShell
+
+# Connect to AzAccount
+# Note: Tenant Id *must* be specified when connecting to AzAccount
+
+$TenantId = "<TenantId>"
+Connect-AzAccount -Tenant $TenantId
+```
+
+Setup the remediation user-assigned MI using Set-AzTSMMARemovalUtilitySolutionRemediationIdentity command. When this command is run successfully, the remediation user-assigned MI is created and the created identity is granted with 'Reader' and 'Virtual Machine Contributor' permissions on target scopes.
+
+``` PowerShell
+# -----------------------------------------------------------------#
+# Step 1: Create remediation user-assigned MI
+# -----------------------------------------------------------------#
+
+$Identity = Set-AzTSMMARemovalUtilitySolutionRemediationIdentity ` 
+         -SubscriptionId <MIHostingSubId> `
+         -ResourceGroupName <MIHostingRGName> `
+         -UserAssignedIdentityName <MIName> `
+         -Location <Location> `
+         -TargetSubscriptionIds @("<SubId1>","<SubId2>","<SubId3>") `
+         -TargetManagementGroupNames @("<MGName1>","<MGName2>","<MGName3>") `
+         -TenantScope
+
+# -----------------------------------------------------------------#
+# Step 2: Save resource id of the identity created using the below command. This will be used in AzTS MMA Removal Utility installation. 
```

Comment:
This will be used during the installation of the AzTS MMA Removal Utility.

Reply:
Addressed in all the places.